src/grpc/dp_grpc_impl.c

```diff
@@ -502,6 +502,12 @@ static int dp_process_create_interface(struct dp_grpc_responder *responder)
 		goto route6_err;
 	}
 
+	if (!port->is_pf)
```

Comment:
You only touch create/delete interface calls. What about get and list interface calls ?